openedx/core/djangoapps/edx_global_analytics/tests/test_tasks.py

```diff
@@ -0,0 +1,52 @@
+"""
+Tests for edx_global_analytics application tasks and helper functions.
+"""
+
+from django.test import TestCase
+
+from ..tasks import platform_coordinates
+
+
+class TestInstallationInformation(TestCase):
```

Comment:
I`d rename class for testing platform coordinates only .
Test class contains a lot corresponding tests.